docs/metrics-yaml-format.md

```diff
@@ -103,9 +106,12 @@ observation:
 
 Sematext App Agent supports the following metric data types:
 
-* `gauge`, `long_gauge`: Gauge data type. Default aggregation is `AVG`
-* `counter`, `long_counter`: Counter data type. Default aggregation is `SUM` 
-* `text`: Textual data type. Typically used for metrics that have to be extracted as tags
+* `gauge`, `long_gauge`:  Gauge is a metric that represents a single numerical value that can arbitrarily go up and down.
+   Examples for gauge metric are current memory usage, current thread count, etc. Default aggregation is `AVG`
+* `counter`, `long_counter`: Counter is a cumulative metric that represents a single monotonically increasing counter 
```

Comment:
Should there be a period (end of sentence) before `long_counter`?

Reply:
Yes. I assume you meant this -  Default aggregation is `AVG`.